web-app/src/store/modules/metakg.js

```diff
@@ -10,20 +10,24 @@ cytoscape.use(popper);
 
 export const metakg = {
     state: () => ({ 
-        "useAPI": true,
+        "baseURL": 'http://localhost:8000/api/metakg/consolidated',
```

Comment:
@marcodarko is it OK to hard-code this URL to `localhost:8000` here? Maybe you update it later, if so, a comment is needed to explain how it works.

Reply:
Actually this needs to be changed to /api/metakg.... for prod as it's the same server taking care of the API and dist webapp, but in local dev I need the specific port since I'm only running the web-app with nodeJS and the api is running separately.  I will make a change and document it.